hphp/hack/src/errors/nast_check_error.ml

```diff
@@ -445,7 +446,29 @@ let illegal_context pos name =
         Naming_special_names.Coeffects.contexts )
     []
 
-let case_fallthrough switch_pos case_pos =
+let case_fallthrough switch_pos case_pos next_pos =
+  let quickfixes =
+    match next_pos with
+    | None -> []
+    | Some next_pos ->
+      let (_, start_col) = Pos.line_column next_pos in
+      let offset = String.length "case " in
+      let new_pos =
+        Pos.set_col_end (start_col - offset)
+        @@ Pos.set_col_start (start_col - offset) next_pos
+      in
+      let new_text =
+        "  // FALLTHROUGH\n" ^ String.make (start_col - offset) ' '
+      in
+
+      [
+        Quickfix.make
+          ~title:"Annotate fallthrough case with // FALLTHROUGH comment"
```

Comment:
This message feels a little like it's focusing on the implementation details of adding a comment. How about this?